src/internal/builtInServices.ts

```diff
@@ -129,6 +129,16 @@ export function builtInServices(context: BuiltInServiceContext): {
             },
         },
 
+        debug: {
+            stringify: (req) => {
+                let arg = req.args[0];
+                if (arg === undefined) {
+                    arg = '';
```

Comment:
Let's return `<empty argument list>` if `req` is empty.

And if `req` has more than 1 element, let's display it as JSON list

Reply:
done